Challenges/challenge_10_making_a_loading_bar/greghilston/src/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>repl.it</title>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+    <label for="secondsinput">Load Time (in seconds)</label>
+    <input id="secondsinput" type="number" min="0" value="3"><br>
```

Comment:
I did the same mistake too, but `input`  props `min`, and `max` are not strictly enforced. I haven't used these properties much, but I'm sure it's only for form submissions. For example, if you input `-5`, the browser won't allow a form `HTTP POST` submission.